android/src/main/java/com/dylanvann/fastimage/FastImageViewWithUrl.java

```diff
@@ -1,14 +1,23 @@
 package com.dylanvann.fastimage;
 
 import android.content.Context;
-
 import androidx.appcompat.widget.AppCompatImageView;
 
+import com.bumptech.glide.integration.webp.decoder.WebpDrawable;
 import com.bumptech.glide.load.model.GlideUrl;
 
 class FastImageViewWithUrl extends AppCompatImageView {
     public GlideUrl glideUrl;
 
+    public void playAnimation() {
+        if (this.getDrawable() instanceof WebpDrawable) {
+            WebpDrawable drawable = (WebpDrawable) this.getDrawable();
+            if (!drawable.isRunning()) {
```

Comment:
I don't remember the exact reason but we were getting some crashes when trying to start a running animation and this check wasn't enough: https://picnic.atlassian.net/browse/PSF-12306
Probably John remembers more about it, but good to keep an eye.

Reply:
Yeah the crash happens when the same drawable is seen on multiple screens, for some reason `isRunning` was not enough
just to be safe we can call `drawable.stop()` before calling `drawable.startFromFristFrame()`